src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -170,12 +171,34 @@ else if(messageContent != null) {
 
       messageStore.addMessage(message);
 
-      List<String> messageInformation = new ArrayList<>();
+      List<String> messageInformation = new ArrayList<String>();
       messageInformation.add(user.getName());
       messageInformation.add(conversationTitle);
       messageInformation.add(cleanedMessageContent);
       Event messageEvent = new Event(UUID.randomUUID(), "Message", message.getCreationTime(), messageInformation);
       eventStore.addEvent(messageEvent);
+
+      // Scan the message for "@NemoBot"
+      if (cleanedMessageContent.toLowerCase().contains("@NemoBot".toLowerCase())) {
```

Comment:
We don't just want this to be contained in the message, we need it to be a standalone word right? Something like "@NemoBotzzz" shouldn't trigger the bot.

Reply:
Right. Would requiring the user to @ mention the bot as the first word in the message be a good way to go about this? That way it'll be simpler to check the message until the first white space?

Reply:
I'd say that's a reasonable initial restriction. One we might decide to relax later on, but good enough for now.